### PR TITLE
Enable e2e capability tests

### PR DESCRIPTION
**Pull request by MiroslavGatsanoga**
# Description

Implement sending E2E test notifications to monitoring subscribers.

## What

Sending the E2E test notifications to monitoring subscribers is done according to a configurable allow list of UUIDs.
The checks that PAM schedules regarding the Push notifications capability now produce metrics indicating success i.e. status metric with value 1 or time metric lower than 120 (our SLA).
The metrics can be found at (https://graphitev2-api.ft.com/) under `Metrics/content-metadata/capability-monitoring directory`.

## Why

[JIRA ticket](https://financialtimes.atlassian.net/browse/UPPSF-1917). 

## Scope and particulars of this PR (Please tick all that apply)

- [ ] Tech hygiene (dependency updating & other tech debt)
- [ ] Bug fix
- [X] Feature
- [ ] Documentation
- [ ] Breaking change
- [ ] Minor change (e.g. fixing a typo, adding config)

___
This Pull Request follows the rules described in our [Pull Requests Guide](https://github.com/Financial-Times/upp-docs/tree/master/guides/pr-guide)

### CODE REVIEW

**Review by ivan-p-nikolov — DISMISSED** [review]
Looks good to me. I would have tried to extract the content UUID from the message body instead of the transaction ID, but maybe that's more work for not a lot of gain.

Can we document in the README `Monitoring` section the new functionality? The fact that we have UUIDs that would not generate notifications for normal subscribers, only monitor ones. Also that the UUID is extracted from the transaction and not the body.

File: consumer/publish_event.go [hunk 1]
```diff
@@ -19,6 +19,18 @@ func (msg NotificationQueueMessage) HasSynthTransactionID() bool {
 	return strings.HasPrefix(tid, "SYNTH")
 }
 
+// HasE2ETestTransactionID checks if the message is e2e test
+func (msg NotificationQueueMessage) HasE2ETestTransactionID(e2eTestUUIDs []string) bool {
+	tid := msg.TransactionID()
+	for _, testUUID := range e2eTestUUIDs {
+		if strings.Contains(tid, testUUID) {
```
Comment (ivan-p-nikolov) [inline consumer/publish_event.go:26]: I was wondering. Should we handle lower/upper case letters for the UUIDs? Currently only metadata messages have upper case letters, but still.

**Review by ivan-p-nikolov — APPROVED** [review]

File: consumer/publish_event.go (hunk 1 above)
Comment (MiroslavGatsanoga) [reply consumer/publish_event.go:26]: I don't see the need for that now but it can be done in the future.

**Review by mitagg — APPROVED** [review]

**Comment by MiroslavGatsanoga** [comment]
Rebased onto master, I think it's ok to disregard the Snyk check fail as I suspect it is an issue with Snyk.

**Review by mitagg — APPROVED** [review]
We can deal with Snyk later, LGTM!

**Review by ivan-p-nikolov — APPROVED** [review]

**Pull request merged**